src/objdump/parser.cpp

```diff
@@ -27,7 +29,7 @@ AsmParser::ObjDumpParser::ObjDumpParser(const Filter filter) : filter(filter)
 
 bool AsmParser::ObjDumpParser::shouldIgnoreFunction(const std::string_view name) const
 {
-    if (auto match = ctre::match<"^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|"
+    if (auto match = ctre::match<"^(__[^_]*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|"
```

Comment:
not sure this is a wise regex change, but this was needed to test `____C_A_T_C_H____T_E_S_T____0():`